plugin/http/httptrace/httptrace.go

```diff
@@ -86,8 +86,8 @@ func NewTransport(base http.RoundTripper, p ...propagation.HTTPFormat) http.Roun
 // The span will be automatically ended by the handler.
 //
 // Incoming propagation mechanism is determined by the given HTTP propagators.
-func NewHandler(base http.Handler, p ...propagation.HTTPFormat) http.Handler {
-	return &handler{handler: base, propogators: p}
+func NewHandler(base http.Handler, format ...propagation.HTTPFormat) http.Handler {
+	return &handler{handler: base, propogators: format}
```

Comment:
shouldn't `propagators` field of handler struct be renamed too, for consistency?

Reply:
Done.